client/dashboard/.eslintrc.js

```diff
@@ -7,9 +7,12 @@ module.exports = {
 					{
 						group: [
 							'calypso/*',
-							// Allowed: calypso/lib/wp
+							// Allowed:
+							// - calypso/lib/url
+							// - calypso/lib/wp
 							'!calypso/lib',
 							'calypso/lib/*',
+							'!calypso/lib/url',
```

Comment:
Should we limit this to `url/http-utils` file for now?
I would love a follow-up to clean the types there (avoid importing calypso/types) to keep the library generic.

Reply:
Done limiting to `url/http-utils`.